student/utils/studentList.js

```diff
@@ -35,20 +35,24 @@ app.controller("myCtrl", function($scope, $filter,$http) {
 		var Batch;
 		var Studentname = $scope.addname;
 		var BatchId = $scope.addbatch;
+		var Email = $scope.addemail;
```

Comment:
Add null checks for email and phone. Verify fees is present as this field is mandatory